client/my-sites/people/people-list-section-header/index.jsx

```diff
@@ -24,7 +24,7 @@ import isSiteAutomatedTransfer from 'state/selectors/is-site-automated-transfer'
 
 class PeopleListSectionHeader extends Component {
 	static propTypes = {
-		label: PropTypes.string.isRequired,
+		label: PropTypes.string,
```

Comment:
Why was this change needed?

Reply:
@Umangvaghela you mentioned above that the `label` prop can be an array. If it can be _either_ an array or a string, we probably want something like:

```label: PropTypes.oneOfType([
    PropTypes.string,
    PropTypes.array,
  ]).isRequired
```

Reply:
@bluefuton 
You want to say me to change  label: PropTypes.string.isRequired, to  label: PropTypes.oneOfType([
    PropTypes.string,
    PropTypes.array,
]).isRequired 

Am I right. 

But In this condition : 
if ( this.props.site && this.props.users.length ) {
			if ( this.props.search && this.props.totalUsers ) {
				headerText = this.props.translate(
					'%(numberPeople)d Person Matching {{em}}"%(searchTerm)s"{{/em}}',
					'%(numberPeople)d People Matching {{em}}"%(searchTerm)s"{{/em}}',
					{
						count: this.props.users.length,
						args: {
							numberPeople: this.props.totalUsers,
							searchTerm: this.props.search,
						},
						components: {
							em: <em />,
						},
					}
				);
				
			}
Var headerText is given array value. That's why the issue is happening.

You want to change only PropTypes.oneOfType([
    PropTypes.string,
    PropTypes.array,
]).isRequired  or headerText  = headerText [0] or  both.

Thanks

Reply:
Howdy @Umangvaghela. Check out [article about creating codeblocks](https://help.github.com/articles/creating-and-highlighting-code-blocks/). It'll make reading code easier for everyone. :-) You can modify your comments to add those blocks. Thanks!